dm/dm/master/shardddl/optimist.go

```diff
@@ -118,9 +118,14 @@ func (o *Optimist) Locks() map[string]*optimism.Lock {
 }
 
 // ShowLocks is used by `show-ddl-locks` command.
```

Comment:
the file permission has been changed

Reply:
addressed in https://github.com/pingcap/tiflow/pull/4087/commits/96b2db76737e4e6221b5f0413690e0be2d8a5e15